### PR TITLE
fix(calendar): add rule on picasso calendar styles component

### PR DESCRIPTION
**Pull request by yonathan-pineda**
[SDE-462](https://toptal-core.atlassian.net/browse/SDE-462) :bug: 

### Description

On version `26.7.0` of Picasso there were implemented a couple of new props for `DatePicker`, a rule (today) on `styles.ts` file of `Calendar` component was removed but it was still being referenced, in results, on staff-portal jest tests were failing because of this lack.

### How to test

- On Staff-Portal run `yarn test libs/filters/src/components/Filters/__tests__/date-range-filter.test.tsx`
- The tests are going to fail with Warning message `Warning: [JSS] Could not find the referenced rule "today" in "PicassoCalendar"`

### Screenshots
picasso-calendar

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

**Breaking change**

- [N/A] codemod is created and showcased in the changeset
- [N/A] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

**Comment by yonathan-pineda** [comment]
@toptal-anvil ping reviewers
@toptal-bot run required

**Review by dwayhs — APPROVED** [review]

**Review by dmaklygin — APPROVED** [review]

**Review by TomasSlama — APPROVED** [review]

**Review by pedrobarrostech — APPROVED** [review]

File: .changeset/shiny-pigs-confess.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@toptal/picasso': patch
+---
+
+add rule on calendar styles
```
Comment (LashaJini) [inline .changeset/shiny-pigs-confess.md:5]:
```suggestion
---
'@toptal/picasso': patch
---

---
### Calendar

- add rule on calendar styles
```

Please follow our [changeset guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md#summary-of-the-change-changelog-message)

**Review by mateusbzerra — APPROVED** [review]

**Comment by yonathan-pineda** [comment]
@toptal-anvil ping reviewers

**Comment by yonathan-pineda** [comment]
@toptal-anvil ping reviewers

**Comment by yonathan-pineda** [comment]
@toptal-anvil ping reviewers

**Review by LashaJini — APPROVED** [review]

**Pull request merged**